Makefile

```diff
@@ -242,9 +242,9 @@ generate-exp-etcdrestore-manifests-api: controller-gen ## Generate ClusterRole a
 	$(CONTROLLER_GEN) rbac:roleName=manager-role crd paths="./exp/etcdrestore/api/..." \
 			paths=./exp/etcdrestore/controllers/... \
 			paths=./exp/etcdrestore/webhooks/... \
-			output:crd:artifacts:config=./config/exp/etcdrestore/crd/bases \
-			output:rbac:dir=./config/exp/etcdrestore/rbac \
-			output:webhook:dir=./config/exp/etcdrestore/webhook \
+			output:crd:artifacts:config=./exp/etcdrestore/config/crd/bases \
+			output:rbac:dir=./exp/etcdrestore/config/rbac \
+			output:webhook:dir=./exp/etcdrestore/config/webhook \
```

Comment:
There is also CRD generation on line 526 which uses config/exp path, which needs to be changed https://github.com/rancher/turtles/pull/689/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R526

Reply:
Yes, exactly. Also please run `make generate`